libraries/botbuilder/tests/streaming/tokenResolver.test.js

```diff
@@ -44,8 +46,10 @@ describe(`TokenResolver`, function () {
     this.timeout(50000000);
```

Comment:
This jumps out at me - necessary?

Reply:
My guess it was used for troubleshooting. I'll play around with it and see if we can lower it.

Reply:
Aside from that this is ready to merge. And feel free to merge this once tests pass and resolve that some other time. We have manual timeouts all over the place, it probably warrants a separate pass at some point.